### ISSUE TITLE
Suggestions not getting displayed

### ISSUE THREAD

**Issue by mishrarohit** [mcrowe/soulmate.js#9]
Hi, 

Thanks for your work on soulmate. On entering any query in the autocomplete box, I do not see any autocomplete suggestions.  I cloned the demo and I can see valid JSON response from Seatgeek, but the suggestions are not being displayed. I ran the `spec/spec_runner.html` file and all specs are passing. 

I am using Chrome Version 21.0.1180.57

**Comment by mishrarohit** [comment]
This is working pretty well in Firefox though.  I am quite confused.

**Comment by mcrowe** [comment]
Hi @movingahead . I just tried the demo in Chrome 21.0.1180.89, and the suggestions are coming up OK. Unfortunately, I'm not sure what might be going wrong in your case. Are there any errors in the console? Can you inspect the source and determine whether the markup for the suggestions is being created, but is just not visible.

**Comment by mishrarohit** [comment]
I am not able to reproduce the issue now. Though, the console shows me this error for every query 
`Uncaught TypeError: Property 'jQuery16407736390051431954_1347530059280' of object [object Window] is not a function autocomplete:1
(anonymous function)`

**Comment by mcrowe** [comment]
That function, I believe, is the jquery jsonp callback. Soulmate.js is telling soulmate to use jsonp so that it can have soulmate running on a different domain. Unfortunately, I'm really not sure why you would be getting that error.

**Comment by thomasklemm** [comment]
I've seen a similar error while using [Swiftype Autocomplete](https://github.com/swiftype/swiftype-autocomplete-jquery). Luke Frankl from Swiftype reported back: "This occurs because the current request for autocomplete results is aborted if the user continues to type before the results have been returned. Canceling the request causes the generated callback to not exist any more."

**Comment by mcrowe** [comment]
@thomasklemm You're right, that may explain the error @movingahead was seeing. Thanks for commenting!

**Comment by liurida** [comment]
Hi, right now, the soulmate can return the search result. But I'm not sure how can I display the json result in the webpage. The script is here:

<script type="text/javascript">
    (function() {
      var render, select;
      $('#search-input').focus();
      render = function(term, data, type) {
         var link = "<a href='#'>"+term+"</a>";
         return link;
      };
      select = function(term, data, type) {
        return console.log("Selected " + term);
      };
      $('#search-input').soulmate({
        url: location.origin+'/ecs/19/auto_complete_by_dvp',
        types: ['name'],
        renderCallback: render,
        selectCallback: select,
        minQueryLength: 2,
        maxResults:     10
      });
    }).call(this);
</script>

My question is how should I define the render and select to make the suggestion display.

**Comment by mcrowe** [comment]
Hi liurida. My apologies. I'm on a long term cycle tour right now, so I'm not really able to look into your question. Hopefully someone in the community can for me.
